src/base/RefCount.h

```diff
@@ -27,6 +27,10 @@ class RefCount
 {
 
 public:
+    template<typename... Args>
```

Comment:
Just to document the new function template. We do not have to do that if you think it is obvious:

```suggestion
    /// creates a new C object using given C constructor arguments (if any)
    /// \returns a refcounting pointer to the created object
    template<typename... Args>
```